src/NServiceBus.AcceptanceTests/Core/FakeTransport/ProcessingOptimizations/When_using_concurrency_limit.cs

```diff
@@ -31,7 +31,10 @@ class ThrottledEndpoint : EndpointConfigurationBuilder
         {
             public ThrottledEndpoint()
             {
-                EndpointSetup<DefaultServer>(c => c.UseTransport<FakeTransport>());
+                EndpointSetup<DefaultServer>(c => {
+                    c.UseTransport<FakeTransport>();
```

Comment:
Note that this is "a different" fake transport, we could use the "main" with some modifications but figured it was not in scope